Containers/domaincheck/Dockerfile

```diff
@@ -1,18 +1,15 @@
 FROM alpine:3.17.3
-RUN apk add --no-cache lighttpd bash netcat-openbsd
-
-RUN adduser -S www-data -G www-data
-RUN rm -rf /etc/lighttpd/lighttpd.conf
-COPY lighttpd.conf /etc/lighttpd/lighttpd.conf
-RUN chmod +r -R /etc/lighttpd && \
+RUN apk add --no-cache bash lighttpd netcat-openbsd; \
+    adduser -S www-data -G www-data; \
+    rm -rf /etc/lighttpd/lighttpd.conf; \
+    chmod +r -R /etc/lighttpd && \
     chown www-data:www-data -R /var/www && \
-    chown www-data:www-data /etc/lighttpd/lighttpd.conf
+    mkdir -p /var/www/domaincheck
```

Comment:
```suggestion
    chmod +r -R /etc/lighttpd; \
    mkdir -p /var/www/domaincheck; \
    chown www-data:www-data -R /var/www
```

Reply:
can you please apply this suggestion, because github reports me "Applying suggestions on deleted lines is currently not supported."

Reply:
done

Reply:
thanks